### ISSUE TITLE
GPAUDITLOGGING-80 Add support for enabling/disabling auto import of AuditLogEvent domain class in HQL queries.

### ISSUE THREAD

**Issue by robertoschwald** [robertoschwald/grails-audit-logging-plugin#90]
opened by Salvador Campos 20/May/15 4:08 PM

I just upgraded from plugin version 1.0.3 to 1.0.4, I need to execute HQL queries on the AuditLogEvent domain class.
Example query
```groovy
def a = AuditLogEvent.executeQuery("FROM AuditLogEvent", [max:10])
```

This caused the following exception
```
org.hibernate.hql.ast.QuerySyntaxException: AuditLogEvent is not mapped [FROM AuditLogEvent]
```

After taking a look to AuditLogEvent.groovy I found this statement on the 
static mapping closure:
```
autoImport false
```

I searched in the grails plugin page but I found nothing about this change, for now I need to use the AuditLogEvent fully qualified name to make the queries work:
```
def a = AuditLogEvent.executeQuery("FROM org.codehaus.groovy.grails.plugins.orm.auditable.AuditLogEvent", [max:10])
It would be nice if this option is also configurable via Config.groovy
```

**Comment by robertoschwald** [comment]
With #13 we will hand over the generation of the AuditLogEvent domain class to the developers, as making every sort of config to this class configurable by the plugin is a pita. With release 1.1.0, everyone has full control of this domain class.

**Comment by robertoschwald** [comment]
Published 1.1.0-SNAPSHOT with externalized AuditLogEvent domain class (see #13 ) support. Would be great if you can test it.
